deploy/buildspec/build-app.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.2
+
+env:
+  shell: bash
+  secrets-manager:
+    DOCKERHUB_USERNAME: "dockerhub:username"
+    DOCKERHUB_TOKEN: "dockerhub:token"
+    VITE_GOOGLE_CLIENT_ID: "rope:googleclientid"
```

Comment:
@P-Gill97 , @MReyna12 - Just as FYI: The client ID used in production is being piped into the FE code here (see note [here](https://github.com/openstax/k12-infra/tree/k12-594/add-rope-deployment/prod/raise/rope#updating-secrets) as far as the JSON structure of the secret in AWS).